lib/kubeclient/common.rb

```diff
@@ -109,6 +109,10 @@ def self.define_entity_methods(entity_types)
         define_method("update_#{entity_name}") do |entity_config|
           update_entity(entity_type, entity_config)
         end
+
+        define_method("patch_#{entity_name}") do |name, namespace = nil, patch|
```

Comment:
@zakiva I agree with @moolitayer that it's better to move the namespace to be the last parameter.
it's also more logical from the perspective of flow: "what to patch, what is the patch content and in which namespace"